src/plugins/tools/apipTextToSpeech/plugin.js

```diff
@@ -316,6 +318,10 @@ export default pluginFactory({
 
                 getTTSComponent().setMediaContentData(ttsApipData);
                 this.show();
+                if (!this.getState('sleep')) {
+                    this.setState('enabled', true);
```

Comment:
Why not use `this.enable()` instead? The state `enabled` is managed internally by the methods `enable()` and `disable()` (cf. [the plugin abstraction](https://github.com/oat-sa/tao-core-sdk-fe/blob/8cf13ba0c121d845d44c6e675084920cabacf6ef/src/core/plugin.js#L325:L344)).

Reply:
there is `enable` method relates to button, not to plugin